Assets/MRTK/Providers/LeapMotion/LeapMotionDeviceManager.cs

```diff
@@ -125,6 +128,34 @@ public override void Enable()
                 // If the leap controller is mounted on a headset then add the LeapXRServiceProvider to the scene
                 // The LeapXRServiceProvider can only be attached to a camera 
                 LeapMotionServiceProvider = CameraCache.Main.gameObject.AddComponent<LeapXRServiceProvider>();
+
+                LeapXRServiceProvider leapXRServiceProvider = LeapMotionServiceProvider as LeapXRServiceProvider;
+
+                // Allow modification of VR specific offset modes if the leapControllerOrientation is Headset
+                // These settings mirror the modification of the properties exposed in the inspector within the LeapXRServiceProvider attached
+                // to the main camera
+                if (leapVRDeviceOffsetMode == LeapVRDeviceOffsetMode.ManualHeadOffset)
+                {
+                    // Change the offset mode before setting the properties 
+                    leapXRServiceProvider.deviceOffsetMode = LeapXRServiceProvider.DeviceOffsetMode.ManualHeadOffset;
+
+                    leapXRServiceProvider.deviceOffsetYAxis = SettingsProfile.LeapVRDeviceOffsetY;
+                    leapXRServiceProvider.deviceOffsetZAxis = SettingsProfile.LeapVRDeviceOffsetZ;
+                    leapXRServiceProvider.deviceTiltXAxis = SettingsProfile.LeapVRDeviceOffsetTiltX;
+                }
+                else if (leapVRDeviceOffsetMode == LeapVRDeviceOffsetMode.Transform)
+                {
+                    if (SettingsProfile.LeapVRDeviceOrigin != null)
+                    {
+                        leapXRServiceProvider.deviceOffsetMode = LeapXRServiceProvider.DeviceOffsetMode.Transform;
+
+                        leapXRServiceProvider.deviceOrigin = SettingsProfile.LeapVRDeviceOrigin;
+                    }
+                    else
+                    {
+                        Debug.LogError("The LeapDeviceOrigin Transform was not set in the LeapMotionDeviceManagerProfile and is null");
```

Comment:
Can this explicity use the term 'Leap VR Device Origin' to be clear exactly what field needs to be set?